### PR TITLE
Add strongnumber.py

### PR DESCRIPTION
**Pull request by Priyanshu729**
in an easy manner

### CODE REVIEW

**Comment by tasnimzotder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- python strongnumber.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935&bid=20439319)

File: python strongnumber.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+def strongnumber(n,sum=0):
```
Comment (tasnimzotder) [inline python strongnumber.py:1]: Codacy found an issue: [Redefining built-in 'sum'](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

File: python strongnumber.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    r=0
+    while n!=0:
+        r=n%10
+        fact=1
+        while r!=0:
+            fact=fact*r
+            r=r-1
+        sum=sum+fact
```
Comment (tasnimzotder) [inline python strongnumber.py:9]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

File: python strongnumber.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+            r=r-1
+        sum=sum+fact    
+        n=n//10
+    return sum
+N=int(input("enter the number"))
+f=strongnumber(N)
+if N==f:
+    print("the number is strongnumber")
+else:
+    print("not")
```
Comment (tasnimzotder) [inline python strongnumber.py:17]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

**Comment by tasnimzotder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- python strongnumber.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935&bid=20439319)

File: python strongnumber.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+def strongnumber(n):
+    sum=0
```
Comment (tasnimzotder) [inline python strongnumber.py:2]: Codacy found an issue: [Redefining built-in 'sum'](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

File: python strongnumber.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    r=0
+    while n!=0:
+        r=n%10
+        fact=1
+        while r!=0:
+            fact=fact*r
+            r=r-1
+        sum=sum+fact
```
Comment (tasnimzotder) [inline python strongnumber.py:10]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

File: python strongnumber.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+            r=r-1
+        sum=sum+fact    
+        n=n//10
+    return sum
+N=int(input("enter the number"))
+f=strongnumber(N)
+if N==f:
+    print("the number is strongnumber")
+else:
+    print("not")
```
Comment (tasnimzotder) [inline python strongnumber.py:18]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

**Comment by tasnimzotder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- python strongnumber.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935&bid=20439319)

File: python strongnumber.py [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+        sum=sum+fact    
+        n=n//10
+    return sum
+
+N=int(input("enter the number"))
+f=strongnumber(N)
+if N==f:
+    print("the number is strongnumber")
+else:
+    print("not")
```
Comment (tasnimzotder) [inline python strongnumber.py:21]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

File: python strongnumber.py [hunk 8]
```diff
@@ -0,0 +1,21 @@
+def strongnumber(n):
+    sum=0
```
Comment (tasnimzotder) [inline python strongnumber.py:2]: Codacy found an issue: [Redefining built-in 'sum'](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

File: python strongnumber.py [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+    r=0
+    while n!=0:
+        r=n%10
+        fact=1
+
```
Comment (tasnimzotder) [inline python strongnumber.py:7]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

File: python strongnumber.py [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+    r=0
+    while n!=0:
+        r=n%10
+        fact=1
+        
+        while r!=0:
+            fact=fact*r
+            r=r-1
+            
+        sum=sum+fact
```
Comment (tasnimzotder) [inline python strongnumber.py:12]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

**Comment by tasnimzotder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- python/strongnumber.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935&bid=20439319)

**Comment by Priyanshu729** [comment]
what do u  mean by trailing spaces

File: python/strongnumber.py [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+    _sum=0
+    r=0
+    while n!=0:
+        r=n%10
+        fact=1
+        
+        while r!=0:
+            fact=fact*r
+            r=r-1          
+        _sum=_sum+fact
```
Comment (tasnimzotder) [inline python/strongnumber.py:11]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

File: python/strongnumber.py [hunk 12]
```diff
@@ -0,0 +1,20 @@
+def strongnumber(n):
+    _sum=0
+    r=0
+    while n!=0:
+        r=n%10
+        fact=1
+
```
Comment (tasnimzotder) [inline python/strongnumber.py:7]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

File: python/strongnumber.py [hunk 13]
```diff
@@ -0,0 +1,20 @@
+def strongnumber(n):
+    _sum=0
+    r=0
+    while n!=0:
+        r=n%10
+        fact=1
+        
+        while r!=0:
+            fact=fact*r
+            r=r-1
```
Comment (tasnimzotder) [inline python/strongnumber.py:10]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

**Comment by tasnimzotder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- python/strongnumber.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935&bid=20439319)

File: python/strongnumber.py [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+    
+    while n!=0:
+        r=n%10
+        fact=1
+        
+        while r!=0:
+            fact=fact*r
+            r=r-1
+            
+        _sum=_sum+fact
```
Comment (tasnimzotder) [inline python/strongnumber.py:13]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

File: python/strongnumber.py [hunk 15]
```diff
@@ -0,0 +1,22 @@
+def strongnumber(n):
+    _sum=0
+    r=0
+
```
Comment (tasnimzotder) [inline python/strongnumber.py:4]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)

**Comment by tasnimzotder** [comment]
image

**Review by tasnimzotder — APPROVED** [review]
👍

**Comment by Priyanshu729** [comment]
pls tell me the problem of conversion .py program

**Pull request merged**